ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml

```diff
@@ -132,7 +132,7 @@ objects:
         touch /tmp/shared/leased
 
         function release() {
-            local resource_name; resource_name="$( jq .name <<<"${resource}" )"
+            local resource_name; resource_name="$( jq .name <<<"${resource}" --raw-output )"
```

Comment:
nit: Can we use `jq --raw-output .name <<<"${resource}"`?  I'm not used to seeing the option way off on the right after a positional argument and here-string.

Reply:
lol, yeah -- that's insane that it even works